Runbook: clarify MimirIngesterReachingSeriesLimit errors and retries (#9410)
grafana/mimir: docs/sources/mimir/manage/mimir-runbooks/_index.md

### MimirIngesterReachingSeriesLimit

> This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester is reaching the limit.

Reviewer: ```suggestion
This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester reaches that limit.
```

Reviewer: Oh, I see. Then, could we say `is close to reaching the limit` or 'nearly reaches the limit` ?

> The threshold is set at 80%, to give some chance to react before the limit is reached.

Reviewer: ```suggestion
The threshold is set at 80% to give the chance to react before the limit is reached.
```

> Once the limit is reached, writes to the ingester will fail for new series. Appending samples to existing ones will continue to succeed.

Reviewer: ```suggestion
After the limit is reached, writes to the ingester fail for new series. Appending samples to existing ones continue to succeed.
```

Reviewer: Can we avoid using "writes" as a noun? Could we say "write requests" or something else, if it's more accurate. We could then also say "Appending samples to existing requests continue to succeed" for greater clarity.

Reviewer: We avoid using "will" in the docs.

> Note that the error responses sent back to the sender are classed as "server error" (5xx), which should result in a retry by the sender.

Reviewer: classed = classified?
Also, could we say "server errors" for agreement?

> While this situation continues, these retries will stall the flow of data, and newer data will queue up on the sender.

Reviewer: ```suggestion
While this situation continues, these retries stall the flow of data, and newer data queues up on the sender.
```

> If the condition is cleared in a short time, service can be restored with no data loss.

Reviewer: ```suggestion
If the condition is cleared in a short time, service is restored with no data loss.
```

Author: I left this one; it seems more correct to me the way I wrote it.
Maybe if you explain more what you dislike about it I can find another wording.

Author: That is what I meant. Restoring service depends on nothing else going wrong.

> This is different to what happens when the `max_global_series_per_user` is exceeded, which is considered a "client error" (4xx) where excess data is discarded.

Reviewer: ```suggestion
This is different to what happens when the `max_global_series_per_user` limit is exceeded, which is considered a "client error" (4xx). In this case, excess data is discarded.
```
